usr/lib/common/loadsave.c

```diff
@@ -2634,15 +2648,31 @@ CK_RV reload_token_object(STDLL_TokData_t *tokdata, OBJECT *obj)
 
     set_perm(fileno(fp));
 
-    if (fread(header, HEADER_LEN, 1, fp) != 1) {
+    if (fread(header, 5, 1, fp) != 1) {
```

Comment:
Please use a constant instead of 5, e.g. HEADER_COMMON_LEN. And maybe you can do the calculation in another set of constants, e.g.

```
#define HEADER_COMMON_LEN 5
[...]
#define HEADER_PART_LEN (HEADER_LEN + HEADER_PREFIX)
[...]
#define PUB_HEADER_PART_LEN (PUB_HEADER_LEN + HEADER_PREFIX)
```

Reply:
introduced HEADER_COMMON_LEN define.